test/Microsoft.DotNet.ApiDiff.Tests/Diff.Operators.Tests.cs

```diff
@@ -0,0 +1,760 @@
+    }
+
+    #region Exclusions
+
+    [Fact]
+    public void TestExcludeAddedOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static explicit operator int(MyClass value) { throw null; }
+                    }
+                }
+                """,
+                expectedCode: "",
+                hideImplicitDefaultConstructors: true,
+                apisToExclude: ["M:MyNamespace.MyClass.op_Explicit(MyNamespace.MyClass)~System.Int32"]);
+    }
+
+    [Fact]
+    public void TestExcludeModifiedOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static explicit operator int(MyClass value) { throw null; }
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static explicit operator byte(MyClass value) { throw null; }
+                    }
+                }
+                """,
+                expectedCode: "",
+                hideImplicitDefaultConstructors: true,
+                apisToExclude: ["M:MyNamespace.MyClass.op_Explicit(MyNamespace.MyClass)~System.Int32", "M:MyNamespace.MyClass.op_Explicit(MyNamespace.MyClass)~System.Byte"]);
+    }
+
+    [Fact]
+    public void TestExcludeRemovedOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static explicit operator int(MyClass value) { throw null; }
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                    }
+                }
+                """,
+                expectedCode: "",
+                hideImplicitDefaultConstructors: true,
+                apisToExclude: ["M:MyNamespace.MyClass.op_Explicit(MyNamespace.MyClass)~System.Int32"]);
+    }
+
+    // The checked operator isn't being handled by Roslyn, so even when it's not going to show up in the diff,
+    // we try to process it but we end up throwing an exception as it is unrecognized.
+    // It's going to be fixed with https://github.com/dotnet/roslyn/pull/77102
+    [Fact(Skip = "https://github.com/dotnet/roslyn/issues/77101")]
+    public void TestExcludeUnmodifiedOperator()
```

Comment:
@ericstj I implemented API exclusion in the tool. Unfortunately, it is still failing because the API needs to be processed/recognized before determining if it needs to be excluded. I added a skipped test anyway.

Reply:
Note that I added 3 other API exclusion tests above this one, but they are excluding non-checked operators, so those work fine.

Reply:
> processed/recognized before determining if it needs to be excluded

I would hope we could feed the exclusion down into the filter used for genapi.  I think that should avoid trying to generate syntax for the API - since we can check if it's included before producing the syntax (just like we do for internal, for example).

Reply:
Yes, let me look at at that. Looks like I missed something.